client/components/token-field/index.jsx

```diff
@@ -24,12 +25,27 @@ var TokenField = React.createClass( {
 	propTypes: {
 		suggestions: React.PropTypes.array,
 		maxSuggestions: React.PropTypes.number,
-		value: React.PropTypes.array,
 		displayTransform: React.PropTypes.func,
 		saveTransform: React.PropTypes.func,
 		onChange: React.PropTypes.func,
 		tokenStatus: React.PropTypes.func,
-		isBorderless: React.PropTypes.bool
+		isBorderless: React.PropTypes.bool,
+		value: function( props ) {
```

Comment:
Based on [this article](https://facebook.github.io/react/blog/2015/10/07/react-v0.14.html#installation), the `production` environment of React doesn't use the development warnings, which I believe means the `propTypes` warnings.